dockerfile/anaconda-ci/requirements.txt

```diff
@@ -0,0 +1,21 @@
+# pip package install list
+# storing it as an independent file enables Dependabot to bump versions with noise and attention
+# silent updates of pylint and astroid usually cause breakages
+# for format details see https://pip.pypa.io/en/stable/reference/requirements-file-format/
+
+# unit tests of all kinds
+pocketlint
+coverage
+pycodestyle
+dogtail
+rpmfluff
+freezegun
+pytest
+
+# jinja templates
+pyyaml
+jinja2
+
+# pylint and its supporting libs
+#pylint
```

Comment:
Btw. there is a new Pylint release. https://github.com/PyCQA/pylint/releases/tag/v2.15.0

Reply:
Thanks! That works with the new python, so I actually enabled it here with the given version and dropped it from dnf in dockerfile. So the PR now makes sense.